### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JesusWisord**
Adding 3 new scrapers for 

GamePlanet
MixUp
Sears

What is left:

- Solve bug for script to actually ends when scrap finishes.
- Run the script to upload all data to Postgres

### CODE REVIEW

File: scraper/utils/constants.py [hunk 1]
```diff
@@ -141,4 +144,73 @@ class ColombiaGamerConfig(Enum):
...
+    TEST_PRODUCTS = _get_test_products(TEST_PATH, SITE_IDS['MixUp'])
+
+class SearsConfig(Enum):
+    """
+    This enum provides configuration constants for Sears scraping
+    """
+    PRODUCT_URLS = [
+        'https://www.sears.com.mx/categoria/16659/xbox/',
+        # 'https://www.sears.com.mx/categoria/16667/playstation/',
+        # 'https://www.sears.com.mx/categoria/16663/nintendo/',
```
Comment (jcalvarezj) [inline scraper/utils/constants.py:203]: Please remove comments for code on **dev**

File: scraper/utils/spiders.py [hunk 2]
```diff
@@ -66,7 +69,7 @@ def parse_product(self, response):
 
         image_xp = (f'//div[contains(@class, "{OLX.IMG_DIV_CLASS.value}")]//'
                     'img/@src')
-        image = response.urljoin(response.xpath(image_xp).get())
+        image = response.xpath(image_xp).get()
```
Comment (jcalvarezj) [inline scraper/utils/spiders.py:72]: Is this change necessary for the OLX Scraper? 
The absolute path/url stops working for me if I change this to xpath instead of urljoin

File: scraper/utils/constants.py (hunk 1 above)
Comment (JesusWisord) [reply scraper/utils/constants.py]: I have uncomment these lines

File: scraper/utils/spiders.py (hunk 2 above)
Comment (JesusWisord) [reply scraper/utils/spiders.py:72]: In this part I leave just the xpath for the image url
